plugins/connection/docker.py

```diff
@@ -4,7 +4,8 @@
 # (c) 2015, Leendert Brouwer (https://github.com/objectified)
 # (c) 2015, Toshio Kuratomi <tkuratomi@ansible.com>
 # Copyright (c) 2017 Ansible Project
-# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
```

Comment:
This needs to stay, otherwise it's impossible to pass the sanity tests.

Reply:
So, it turns out that `reuse addheader` is not supposed to remove the old license headers; it's a bug. Currently, it completely replaces each "comment block" that contains any copyright/license information with its own formatted version. See https://github.com/fsfe/reuse-tool/issues/338. I came up with an absolutely terrifying incantation to separate `# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)` with a newline so it doesn't get clobbered.